inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -720,8 +722,8 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
             input->getPrecision().size(),
             outputs->getPrecision().size(),
             // TODO: only fp32 and Int16 tested
-            quantized == nullptr ? input->getPrecision().size() : 2,
-            quantized == nullptr ? input->getPrecision().size() : 4,
+            quantized == nullptr ? input->getPrecision().size() : (!gnaFlags->input_low_precision ? 2 : 1),
+            quantized == nullptr ? input->getPrecision().size() : (!gnaFlags->input_low_precision ? 4 : 1),
```

Comment:
consider to simplify the condition by removing `!`

Reply:
Done.